Update BF CF docs (#646)
RedisBloom/RedisBloom: docs/commands/cf.reserve.md

Creates an empty cuckoo filter with a single sub-filter for the initial specified capacity.

> Because of how cuckoo filters work, the filter is likely to declare itself full before `capacity` is reached and therefore fill rate will likely never reach 100%.

Reviewer: ```suggestion
According to the cuckoo filter behavior, the filter is likely to declare itself full before `capacity` is reached; therefore, the fill rate will likely never reach 100 percent.
```

<details open><summary><code>EXPANSION expansion</code></summary>

> When a new filter is created, its size is the size of the current filter multiplied by `expansion` - a non-negative integer. Expansion is rounded to the next `2^n` number. The default value is 1.

Reviewer: ```suggestion
When a new filter is created, its size is the size of the current filter multiplied by `expansion`, specified as a non-negative integer. Expansion is rounded to the next `2^n` number. The default value is `1`.
```
